<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drwSimplifiedToolbar</class>
 <widget class="QWidget" name="drwSimplifiedToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QToolButton" name="smallBrushButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/brush.png</normaloff>:/icons-60x60/brush.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">paintToolButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="eraserButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/eraser.png</normaloff>:/icons-60x60/eraser.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">paintToolButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="drwGradientWidget" name="opacityWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pressureWidthButton">
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/pressure-width.png</normaloff>:/icons-60x60/pressure-width.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pressureOpacityButton">
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/pressure-opacity.png</normaloff>:/icons-60x60/pressure-opacity.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="fillButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/fill.png</normaloff>:/icons-60x60/fill.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="manualModeButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/hand.png</normaloff>:/icons-60x60/hand.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">frameChangeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="jumpModeButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/jump.png</normaloff>:/icons-60x60/jump.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">frameChangeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="playModeButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/play-mode.png</normaloff>:/icons-60x60/play-mode.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">frameChangeButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QSpinBox" name="onionBeforeSpinBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="oneOnionButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lineslibresources.qrc">
       <normaloff>:/icons-60x60/onion.png</normaloff>:/icons-60x60/onion.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="onionAfterSpinBox">
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="lineslibresources.qrc">:/icons-60x60/persistence.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="persistenceSpinBox">
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="netStateLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="lineslibresources.qrc">:/icons-60x60/net-not-connected.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drwGradientWidget</class>
   <extends>QWidget</extends>
   <header>drwGradientWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="lineslibresources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="frameChangeButtonGroup"/>
  <buttongroup name="paintToolButtonGroup"/>
 </buttongroups>
</ui>
